graphics/src/test/kotlin/ktx/graphics/graphicsTest.kt

```diff
@@ -106,6 +106,28 @@ class GraphicsTest {
     verify(batch).end()
   }
 
+  @Test
+  fun `should begin with provided projection matrix`() {
+    val batch = mock<Batch>()
+    val matrix = Matrix4(FloatArray(16) {it.toFloat()})
+
+    batch.begin(projectionMatrix = matrix)
+    verify(batch).projectionMatrix = matrix
+    verify(batch).begin()
+    batch.end()
```

Comment:
Since it's a mocked object either way, cleanup is not necessary.

Reply:
Done